client/src/pages/Dashboard/components/SurveyCard.jsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import {
+  Card,
+  CardActions,
+  CardContent,
+  Button,
+  Box,
+  Chip,
+  Grid,
+  Typography,
+} from '@material-ui/core';
+
+import ProgressWheel from '../../../components/ProgressWheel/ProgressWheel';
+
+import styles from './SurveyCard.module.css';
+
+const SurveyCard = ({ survey }) => {
+  const { _id, title, description, recipients, responses } = survey;
+  const percentage = Math.floor((responses.length / recipients.length) * 100);
```

Comment:
See my comment in `ProgressWheel.jsx` about percentages.

Reply:
done